tests/Unit/Math/BigInteger/TestCase.php

```diff
@@ -193,7 +193,7 @@ public function testBitwiseAND()
         $y = $this->getInstance('33333333333333333333333', 16);
         $z = $this->getInstance('22222222222222222222222', 16);
 
-        $this->assertSame($z->toHex(), $x->bitwise_AND($y)->toHex());
+        $this->assertSame($z->toHex(), $x->bitwise_and($y)->toHex());
```

Comment:
If these are incorrect calls, they should be fixed in the legacy 1.0 branch as well.